app/controllers/application_controller.rb

```diff
@@ -83,6 +83,11 @@ def doorkeeper_unauthorized_render_options(error: nil)
     session[:locale] = nil
     set_locale
 
+    if current_user.should_claim_wca_id?
+      flash[:info] = t('devise.after_sign_in.wca_claim_alert', claim_wca_id_path: profile_claim_wca_id_url).html_safe
```

Comment:
Does this depend on https://github.com/thewca/worldcubeassociation.org/pull/11718?